Userland/Applications/SoundPlayer/SoundPlayerWidgetAdvancedView.cpp

```diff
@@ -64,16 +64,18 @@ SoundPlayerWidgetAdvancedView::SoundPlayerWidgetAdvancedView(GUI::Window& window
     auto& menubar = toolbar_container.add<GUI::Toolbar>();
 
     m_play_button = menubar.add<GUI::Button>();
+    m_stop_button = menubar.add<GUI::Button>();
+
```

Comment:
Moving this isn't really necessary, and I think it makes more sense for the assignment to be next to its `set_foo`'s...